src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -55,13 +57,90 @@ void createStation() {
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철 목록을 조회한다.")
+    @Test
+    void 지하철역_목록_조회() {
+        // given
+        String stationName1 = "교대역";
+        this.createStation(stationName1);
+        String stationName2 = "역삼역";
+        this.createStation(stationName2);
+
+        // when
+        ExtractableResponse<Response> response = this.getStationList();
+
+        // then
+        JsonPath jsonPath = response.jsonPath();
+        List<String> stationNames = jsonPath.getList("name", String.class);
+
+        assertThat(stationNames).hasSize(2);
+        assertThat(stationNames).contains("교대역", "역삼역");
+    }
 
     /**
      * Given 지하철역을 생성하고
      * When 그 지하철역을 삭제하면
      * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
-    // TODO: 지하철역 제거 인수 테스트 메서드 생성
+    @DisplayName("지하철역을 삭제한다.")
+    @Test
+    void 지하철역_삭제() {
+        // given
+        String stationName = "강남역";
+        this.createStation(stationName);
+
+        // when
+        ExtractableResponse<Response> responseAfterStationCreation = this.getStationList();
+        JsonPath jsonPathAfterStationCreation = responseAfterStationCreation.jsonPath();
+        List<Long> stationIds = jsonPathAfterStationCreation.getList("id", Long.class);
+        Long stationId = stationIds.get(0);
+
+        given()
+        .when()
+            .delete("/stations/"+stationId)
```

Comment:
delete 등의 메서드는 path parameter를 지원하니 활용해 보셔도 좋을 것 같아요

```suggestion
            .delete("/stations/{stationsId}", stationId)
```